src/main/java/cloud/eppo/ConfigurationRequestor.java

```diff
@@ -1,55 +1,102 @@
 package cloud.eppo;
 
-import java.io.IOException;
-import okhttp3.Response;
+import cloud.eppo.api.Configuration;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 // TODO: handle bandit stuff
 public class ConfigurationRequestor {
   private static final Logger log = LoggerFactory.getLogger(ConfigurationRequestor.class);
+  public static final String FLAG_CONFIG_PATH = "/api/flag-config/v1/config";
+  public static final String BANDIT_PARAMETER_PATH = "/api/flag-config/v1/bandits";
 
   private final EppoHttpClient client;
-  private final ConfigurationStore configurationStore;
   private final boolean expectObfuscatedConfig;
+  private final boolean supportBandits;
+
+  public ConfigurationRequestor(EppoHttpClient client, boolean expectObfuscatedConfig) {
+    this(client, expectObfuscatedConfig, true);
+  }
 
   public ConfigurationRequestor(
-      ConfigurationStore configurationStore,
-      EppoHttpClient client,
-      boolean expectObfuscatedConfig) {
-    this.configurationStore = configurationStore;
+      EppoHttpClient client, boolean expectObfuscatedConfig, boolean supportBandits) {
     this.client = client;
     this.expectObfuscatedConfig = expectObfuscatedConfig;
+    this.supportBandits = supportBandits;
```

Comment:
This seemed a half-decent way to avoid loading bandit data to the sub-SDKs (android) which don't support bandits at this time. 
If this seems ok, I'll plumb it through the `BaseEppoClient`.

Reply:
📈